scripts/lint-style.lean

```diff
@@ -104,6 +105,21 @@ def allScriptsDocumented : IO Bool := do
       {String.intercalate "," undocumented.toList}"
   return undocumented.size == 0
 
+/-- Verifies that all modules `modules` are named in `UpperCamelCase`
+(except for explicitly discussed exceptions, which are hard-coded here). -/
+def checkModulesUpperCamelCase (modules : Array Lean.Name) : IO Bool := do
+  -- Exceptions to this list should be discussed on zulip!
+  let exceptions := [
+    `Mathlib.Analysis.CStarAlgebra.lpSpace,
+    `Mathlib.Analysis.InnerProductSpace.l2Space,
+    `Mathlib.Analysis.Normed.Lp.lpSpace
+  ]
+  -- We allow only names in UpperCamelCase, possibly with a trailing underscore.
+  let badNames := modules.filter fun name ↦!exceptions.contains name &&
+    (Lake.toUpperCamelCase name != name && s!"{Lake.toUpperCamelCase name}_" != name.toString)
```

Comment:
```suggestion
  let badNames := modules.filter fun name ↦
    let upperCamelName := Lake.toUpperCamelCase name
    !exceptions.contains name &&
    (upperCamelName != name && s!"{upperCamelName}_" != name.toString)
```
just to avoid calling `toCamel` twice.